src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -217,46 +171,152 @@ public void GarbageCollectInstalledWorkloadPacks(DirectoryPath? offlineCache = n
                     else
                     {
                         packsToRemove.Add(packRecord);
+                        Log?.LogMessage($"Removing {packRecord.MsiId} ({packRecord.MsiNuGetVersion}) as no dependents remain.");
```

Comment:
Maybe change the message to `Adding .... for removal as no dependents remain` since this is not actually removing the pack, but adding it to the list of packs to remove

Reply:
Done, good suggestion. Thanks.